axelor-account/src/main/java/com/axelor/apps/account/service/ReconcileGroupServiceImpl.java

```diff
@@ -267,4 +284,36 @@ protected List<Reconcile> getReconcileList(ReconcileGroup reconcileGroup) {
         .bind("confirmed", ReconcileRepository.STATUS_CONFIRMED)
         .fetch();
   }
+
+  @Override
+  public void letterImportedReconcileGroup(FECImport fecImport) throws AxelorException {
+
+    List<Integer> status =
+        Arrays.asList(MoveRepository.STATUS_ACCOUNTED, MoveRepository.STATUS_DAYBOOK);
+    List<ReconcileGroup> reconcileGroups =
+        Beans.get(ReconcileGroupRepository.class)
+            .all()
+            .filter(
+                "self IN (SELECT reconcileGroup FROM MoveLine moveLine WHERE moveLine.move.statusSelect IN :status AND moveLine.move.fecImport = :fecImport)")
+            .bind("status", status)
+            .bind("fecImport", fecImport)
+            .fetch();
+    for (ReconcileGroup reconcileGroup : reconcileGroups) {
+      this.letter(reconcileGroup);
+
```

Comment:
Need to put this code in a new transactional method. Need to manage the jpa cache, as we can have severals reconcileGroup